config/global.ini.php

```diff
@@ -397,6 +397,9 @@
 ; Sets the session cookie path
 login_cookie_path =
 
+; the amount of time before a session that was created without the "remember me" option is considered expired
```

Comment:
Is this from the time the session was created or rather from the last server side requets this user made?

Reply:
the time the last server side request the user made, will update the docs